lang/jerryscript/Makefile

```diff
@@ -0,0 +1,52 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=jerryscript
+PKG_VERSION:=b2f3ec2
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/Samsung/jerryscript.git
+PKG_SOURCE:=$(PKG_NAME).tar.gz
+PKG_SOURCE_VERSION:=b2f3ec225e931e5b2ddac6a249626ffce8fdf045
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)
```

Comment:
Since we already have the commit id in `PKG_SOURCE_VERSION`, we tend to use the commit date for `PKG_VERSION` if that commit id does not correspond to a release version name.

`PKG_SOURCE` needs to be more specific containing version info because the build system will skip making tarballs if it finds there already exists a file with the same name.  `PKG_SOURCE_SUBDIR` can also be more specific.